account_currency_rate_difference_reference/i18n/es_MX.po

```diff
@@ -0,0 +1,47 @@
+# Translation of Odoo Server.
```

Comment:
if you have translation in es.po you don't need es_MX.po

es_MX.po is only needed to translate words that are different from spanish from spain